diff --git a/student.c b/student.c
index 7dc13b5..ff31847 100644
--- a/student.c
+++ b/student.c
@@ -1,9 +1,16 @@
 #include "student.h"
 
+/**
+ * Takes the square of an integer. Assumes that the result will fit into an int.
+ */
 int square(int x) {
   return x * x;
 }
 
+/**
+ * Takes in an integer array and its length, and returns the sum of the elements
+ * of the array. Assumes that the result will fit into an int.
+ */
 int sum(int* array, int length) {
   int running_total = 0;
 
